apps/public-docsite-v9/src/Concepts/BrowserSupportMatrix.stories.mdx

```diff
@@ -18,14 +27,71 @@ FluentUI supports browsers that are at or above the indicated versions below:
+full browser support matrix. For consumers who require the use/support of older browsers, please see the partial support matrix and polyfill sections
+below for more information on what will need to be done.
+
+## Partial Browser Support Matrix:
+
+Fluent UI introduces a new concept of a partial browser support matrix as a means to find that balance between supporting modern browsers and older browsers.
+The partial browser matrix below will have limitations that require consumers to transpile down and/or use polyfills to cover the support gaps. The feature gaps
+are listed below with subsequent recommendation on how to handle each case.
+
+**Desktop Browsers:**
+
+| Edge | Firefox | Chrome | Safari | Opera | Internet Explorer |
+| ---- | ------- | ------ | ------ | ----- | ----------------- |
+| >=79 | >=69    | >=79   | >=13.1 | >=64  | Not Supported     |
+
+**Mobile Browsers:**
+
+| Safari on iOS | Chrome for Android | Samsung |
+| ------------- | ------------------ | ------- |
+| >=13.4        | >=79               | >=14    |
+
+List of features that are currently used in Fluent UI that aren't supported out of the box by the partial support matrix above:
+
+**Unsupported CSS Features:**
+
+- CSS features below are not supported by some browsers within the partial support matrix. The absence of these features will not crash consumer applications,
+  it will simply degrade the user interface.
+
+1.  [Flex gap](https://caniuse.com/flexbox-gap): Unsupported by Chrome, Edge, Firefox, Opera, Safari, Safari on IOS, Chrome on Android
+2.  [CSS min, max, clamp](https://caniuse.com/css-math-functions): Unsupported by Firefox and Opera
+3.  [CSS revert value](https://caniuse.com/css-revert-value): Unsupported by Opera
+
+**Unsupported ECMAScript Features:**
+
+- As previously mentioned, Fluent UI will be targetting ES2020 as its Typescript target. Some ES2020 features are already present
+  in the library and the partial support matrix does not fully support ES2020. This means that consumers who follow the partial support matrix or any browser versions below it
+  will need to transpile down the Fluent UI library to make it work since the use of these ES2020 features results in the application crashing on the listed browsers below.
+
+1. Nullish Coalescing: Unsupported by Chrome, Edge, Firefox, Opera.
+2. Optional Chaining: Unsupported by Chrome, Edge, Firefox, Opera.
+
+## Polyfills
+
+By default Fluent UI will not be providing polyfills for features we expect our full browser support matrix to already support.
+The only instance that a polyfill may be provided is when Fluent UI's use of a feature causes an application to crash on
+the partial support browser matrix. Other than that, consumers will need to implement their own polyfills or use a third party one for their
+applications. Handling polyfills on the application level ensures that those on modern browsers don't suffer from unnecessary overhead of
```

Comment:
this is great explanation. might be good idea to structure this to bullet points so its more "explicit"

```
Other than that, consumers will need to implement their own polyfills or use a third party one for their
applications.

**Why?:**

-  Handling polyfills on the application level ensures that those on modern browsers don't suffer from unnecessary o ...
- prevents any unintended polyfill duplication from the library and app level
- gives consumers the flexibility to provide robust solutions such as shipping a bundle with the necessary polyfills to support older browsers and one with a lighter,
polyfill-free bundle to support modern browsers
```